components/Footer.tsx

```diff
@@ -50,7 +50,7 @@ const Footer: FunctionComponent<FooterProps> = ({
             <h5>{header}</h5>
             <ul className="footer-link-list">
               <FooterLink linkText={imprintLinkText} linkURL={'/imprint' + `?state=${language}`} />
-              <FooterLink linkText={disclaimerLinkText} linkURL="/" />
+              <FooterLink linkText={disclaimerLinkText} linkURL={'/dataPrivacy' + `?state=${language}`} />
               <FooterLink linkText={cookiePolicyLinkText} linkURL="/" />
               <FooterLink linkText={contactLinkText} linkURL="/" />
```

Comment:
Fix that url

Reply:
Could you give me more input on this? The link works fine for me.